prime-router/metadata/fhir_mapping/config.properties

```diff
@@ -0,0 +1,5 @@
+base.path.resource=./metadata/fhir_mapping
+supported.hl7.messages=ADT_A01, ADT_A03, ADT_A08, ADT_A34, ADT_A40, DFT_P03, MDM_T02, MDM_T06, OML_O21, ORM_O01, OMP_O09, ORU_R01, PPR_PC1, RDE_O11, RDE_O25, VXU_V04
+default.zoneid=+08:00
```

Comment:
+8?  Why not UTC like the rest of the code?

Reply:
Good point, this was default from the used library.